content/templates/_index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Template Gallery
-alwaysopen: true
+alwaysopen: false
```

Comment:
This was something that confused me when I was working on the changes to the sidebar.

Live on the site right now, there’s an empty `<ul/>` inside the Template Gallery item. I believe this is because the children (snippets e.g.) are set to always be hidden from the sidebar.

This is producing a small amount of extra vertical space between it and its next sibling.

![image](https://user-images.githubusercontent.com/154613/67887807-60065b80-fb22-11e9-8cf9-afff836a18f0.png)

The contents of the `<ul/>` look like this:

![image](https://user-images.githubusercontent.com/154613/67887845-71e7fe80-fb22-11e9-91c1-d6f7f757a38f.png)

I believe this is because Hugo is looping over the children and rendering a new line character for each child.

Anywho, in working on this change (originally https://github.com/cloudflare/workers-docs/pull/487), I made all of the other top-level sidebar nav items collapsed. My rationale was that even though we currently don’t have any children displayed even when you’re looking at the Template Gallery:

![image](https://user-images.githubusercontent.com/154613/67887963-a9ef4180-fb22-11e9-8745-4750c6e0be2b.png)

...I didn’t know that that would be the case forever. So for future protection I thought it would be safer to collapse it like its siblings.